README.md

```diff
@@ -1,57 +1,18 @@
 # Northcoders House of Games API
 
-## Background
+## Setup
+In order to successfully connect the 2 databases locally, you must add some.env files to your local project.
 
-We will be building an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture.
+### Adding .env files
+Create two files:
+1) .env.development
+    - This will allow connection to your development database
+    - Paste the following into the file:
+        PGDATABASE=nc_games
 
-Your database will be PSQL, and you will interact with it using [node-postgres](https://node-postgres.com/).
+2) .env.test
+    - This will allow connection to your test database
+    - Paste the following into the file:
```

Comment:
Hey Charlie, great start! The only thing we may want to change is the PGDATABASE instructions so that we aren't telling them the name of your database (this could be a bit dangerous and is the reason for us having the env files in our git ignore) Instead is we write PGDATABASE=data_base_name  and PGDATABASE=data_base_name_test. Once this has been corrected feel free to merge and press on with the next ticket :)